src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -34,7 +34,7 @@
     'azure-cli-core',
     'azure-mgmt-web==0.32.0',
     # v1.17 breaks on wildcard cert https://github.com/shazow/urllib3/issues/981
-    'urllib3[secure]==1.16',
+    'urllib3[secure]>=1.18',
```

Comment:
Generally speaking, we shouldn't have `==` in `setup.py` in order to allow the dependencies to patch themselves. With that been said, it will also cause the product be broke by breaking change.